docs/developer/debugging/actors.mdx

```diff
@@ -1,170 +1,100 @@
-to run or fail to properly operate. Below we'll walk through a few of these scenarios with
-accompanying error messages.
+When developing actors, there are a few common errors that can cause your actor to fail to run or properly operate. Below we'll walk through a few of these scenarios with accompanying error messages.
 
-## Common Errors
+### Failing to start an actor from a local registry over HTTP
 
-### Using an insecure registry
+The following error can happen when attempting to start an actor from a local registry server over HTTP: 
 
 ```console
-Failed to fetch OCI bytes: error sending request for url (https://localhost:5000/v2/): error trying to connect: record overflow
-```
-
-This error can happen when attempting to run an actor from the local docker registry included in our examples. If you see this error, you need to set the `WASMCLOUD_OCI_ALLOWED_INSECURE` variable either in your [host config](/docs/hosts/elixir/host-configure) file or as an environment variable. The proper syntax for this is as follows:
+2023-12-19T22:31:26.154791Z ERROR wasmcloud_host::wasmbus: failed to scale actor actor_ref=127.0.0.1:5000/v2/hello:0.1.0 err=failed to fetch actor
 
-<Tabs>
-  <TabItem value="unix" label="Unix" default>
-
-```shell
-export WASMCLOUD_OCI_ALLOWED_INSECURE=localhost:5000
-# Start your host as you did before, either with `mix` for development or the release tarball script
+Caused by:
+    0: failed to fetch actor under OCI reference `127.0.0.1:5000/v2/hello:0.1.0`
+    1: failed to fetch OCI path
+    2: failed to fetch OCI bytes
+    3: error sending request for url (https://127.0.0.1:5000/v2/): error trying to connect: received corrupt message of type InvalidContentType
 ```
 
-  </TabItem>
-  <TabItem value="windows" label="Windows Powershell">
-
-```powershell
-$env:WASMCLOUD_OCI_ALLOWED_INSECURE = localhost:5000
-# Start your host as you did before, either with `mix` for development or the release tarball script
-```
-
-  </TabItem>
-  <TabItem value="docker" label="Docker">
+To fix this error, you need to set the `--allowed-insecure` [argument on the host](/docs/reference/host-config).
 
 :::warning
-When accessing the local registry from _within_ the docker compose network, you should use `registry:5000` instead of `localhost:5000`. Even though port 5000 is available for you on your local machine to interact with the registry, networking works differently within a compose network and you should use the name of the container instead of localhost. We do the same thing with `nats`, as you can see below.
+When accessing a local registry from within a docker compose network, you should use `registry:5000` (or the name of the registry container, if it differs) instead of `localhost:5000`. Even though port 5000 may be available on your host's network, networking works differently within docker, and you should use the name of the container instead of localhost. We do the same thing with `nats`, as you can see below.
```

Comment:
```suggestion
When accessing a local registry from within a docker compose network, you should use `registry:5000` (or the name of the registry container, if it differs) instead of `localhost:5000`. 

While port 5000 may be available on your host's network, default docker compose networking settings often require using the name of the container instead of `localhost`. We follow this practice when referring to `nats` below.
```